### ISSUE TITLE
WOOL Editor - Bug: Background is different between Start/End node and other nodes (when testing dialogue).

### ISSUE THREAD

**Issue by tbeinema** [woolplatform/wool#68]
**Editor Version:**
v1.4.3

**Desktop or Web version:** 
Desktop

**Operating System (Desktop) or Browser (Web):**
Windows

**Describe the Issue:**
When starting the dialogue with the agent the background has one color (Background A), but for the next steps the background has another color (Background B). At the end of the dialogue the color then goes back to the first color again (Background A).

It seems that the Start and End nodes have different backgrounds (Background A) consistently, also when returning to the Start node from a node somewhere else in the conversation. 

E.g.:
1. The start node:
![afbeelding](https://user-images.githubusercontent.com/10047603/83880971-483e2800-a740-11ea-8c9c-c143357449a4.png)
2. Next step:
![afbeelding](https://user-images.githubusercontent.com/10047603/83881046-5e4be880-a740-11ea-907b-f44ad0ecfa23.png)
3. Dialogue is finished:
![afbeelding](https://user-images.githubusercontent.com/10047603/83881120-77ed3000-a740-11ea-8b52-f8f5a7be2d5e.png)

**Comment by borisvanschooten** [comment]
Do the start and end nodes have the same colour as the other nodes?

**Comment by tbeinema** [comment]
No, the 'Start' and 'End' nodes are blue and the other nodes are the standard color. 

I just tried changing the 'Start' and 'End' nodes to the standard color and then everything has the same background. This background is Background B, the one that was shown for all other nodes (non Start or End nodes) previously.
